### ISSUE TITLE
Wrong divider value when converting temperature from LSB to Celsius.

### ISSUE THREAD

**Issue by Shivam1282** [STMicroelectronics/lis2hh12#3]
In **lis2hh12_reg.c** file,

```
float_t lis2hh12_from_lsb_to_celsius(int16_t lsb)
{
  return (((float_t)lsb / 8.0f) + 25.0f);
}
```

In above function, the divider value needs to be changed from **8.0f** to **256.0f** for correct temperature value in celsius.

**Comment by avisconti** [comment]
> In **lis2hh12_reg.c** file,
> 
> ```
> float_t lis2hh12_from_lsb_to_celsius(int16_t lsb)
> {
>   return (((float_t)lsb / 8.0f) + 25.0f);
> }
> ```
> 
> In above function, the divider value needs to be changed from **8.0f** to **256.0f** for correct temperature value in celsius.

Indeed! Thanks for reporting it. I'll add a patch asap.

**Comment by avisconti** [comment]
Resolved on v1.1.1
